components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -714,6 +812,20 @@ - (CGSize)expectedSizeForView:(UIView *)view {
   return expectedItemSize;
 }
 
+- (CGSize)estimatedItemViewSizeForClusteredFixedLayout {
+  CGSize largestSize = CGSizeZero;
+  for (UIView *view in self.itemViews) {
+    CGSize intrinsicContentSize = view.intrinsicContentSize;
+    if (intrinsicContentSize.width > largestSize.width) {
```

Comment:
Personally I think this would be more _readable_ to me if we used `largestWidth` and `largestHeight` for all the comparisons then at the end did `CGSizeMake(largestWidth, largestHeight)`. Just an idea, I'll leave it up to you, if you think this is better feel free to resolve.

Reply:
Done.